jmh-core-it/src/main/java/org/openjdk/jmh/it/CustomExecutorService.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
* Copyright Amazon.com Inc. or its affiliates.

Is that how it should be?

Reply:
Yes, that's our standard header. Deliberately year-less.